src/scss/_functions.scss

```diff
@@ -88,3 +88,26 @@
 		@return $o-typography-display;
 	}
 }
+
+/// Changes a px value to a rem value if relative units are enabled.
+/// Useful for user input such as custom line-heights, where a px value
+/// should become a rem value but otherwise the value should remain unchanged.
+/// @access private
+/// @param {Number} $value
+@function _oTypographyCorrectUnit($value) {
```

Comment:
`oTypographyCorrectUnit` makes it seem like you've done something wrong by passing in a value, maybe we could call it `oTypographyAdjustUnit`?